README.md

```diff
@@ -1,3 +1,5 @@
 This is a repository for random python scripts demonstrating various
 features. The initial collection is made up of scripts created from
 a course in python 3+
+
```

Comment:
What were you thinking!??

Reply:
I was thinking the right thing -- I am always right.